google/cloud/storage/client.py

```diff
@@ -962,9 +962,9 @@ def generate_signed_post_policy_v4(
         :param blob_name: Object name.
 
         :type expiration: Union[Integer, datetime.datetime, datetime.timedelta]
-        :param expiration: Policy expiration time. If expiration in datetime,
-                           then need to pass timezone with it. Default value of
-                           ``tzinfo`` is ``UTC``.
+        :param expiration: Policy expiration time. If a ``datetime`` instance is
+                           passed without an explicit ``tzinfo`` set,  it will be
+                           converted to ``UTC``.
```

Comment:
Per @andrewsg 
```suggestion
                       ``tzinfo`` set,  it will be assumed to be ``UTC``.
```

Reply:
All the changes suggested by @andrewsg has been already pushed. I have verified all the files.

Reply:
@HemangChothani see [this comment by @andrewsg ](https://github.com/googleapis/python-storage/pull/250#discussion_r475949117):

> Rather than "it will be converted to" let's use "it will be assumed to be". This is a huge footgun; a user who sees the term "converted" will assume that we will do the right thing in the default case, whereas in fact we will almost always do the wrong thing.